extras/rxjava/src/main/java/org/asynchttpclient/extras/rxjava/AsyncHttpClientErrorException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015 Target, Inc. All rights reserved.
```

Comment:
I can change that, but can you clarify the policy - many other files have copyright headers for corporations e.g. Sonatype, Ning, Google etc.